src/flask/app.py

```diff
@@ -1276,7 +1276,7 @@ def _find_error_handler(self, e: Exception) -> t.Optional[ErrorHandlerCallable]:
         """
         exc_class, code = self._get_exc_class_and_code(type(e))
 
-        for c in [code, None]:
+        for c in [None] if code is None else [code, None]:
```

Comment:
What about `for c in {code, None}:`? Reads more clearly clearer to me

Reply:
Order isn't guaranteed for sets.

Reply:
Oh yea, the order matters.